packages/frontend/src/features/scheduler/components/SchedulerModal.tsx

```diff
@@ -34,11 +35,21 @@ const SchedulersModal: FC<
                     <Group spacing="xs">
                         <MantineIcon icon={IconBell} size="lg" color="gray.7" />
                         <Text fw={600}>Alerts</Text>
+                        <DocumentationHelpButton
+                            href="https://docs.lightdash.com/guides/how-to-create-alerts"
+                            pos="relative"
+                            top="2px"
+                        />
                     </Group>
                 ) : (
                     <Group spacing="xs">
                         <MantineIcon icon={IconSend} size="lg" color="gray.7" />
                         <Text fw={600}>Scheduled deliveries</Text>
+                        <DocumentationHelpButton
+                            href="https://docs.lightdash.com/guides/how-to-create-scheduled-deliveries"
```

Comment:
Do we think these should open in another window? These only show up in part of the app (this modal) where there is an incomplete form and no saved draft. So if they did already do something in this form, then click that link, they will always loose state. 

@PriPatel any thoughts?

Reply:
Oh woops! yes these should definitely open in a new tab @rephus @magnew

Reply:
this component already opens links in a new tab, we use this component in a few other places . Does it not open in a new tab for you ?

Reply:
Re-tested and this does what I expect. I must have been losing it yesterday. Sorry about the confusion!